capture-sdk/sdk/src/main/res/values-night/styles.xml

```diff
@@ -13,4 +13,28 @@
         <item name="colorOnSurfaceInverse">@color/Dark_02</item>
     </style>
 
+    <style name="GiniCaptureTheme.Analysis" parent="GiniCaptureTheme">
```

Comment:
[Request] Try using the default theme and text styles and override the text colors in the layouts.